jalapeno-helm/templates/collectors/gobmp/collector-gobmp-svc.yaml

```
@@ -1,24 +1,39 @@
 apiVersion: v1
 kind: Service
+metadata:
+  name: {{ .Values.collectors.gobmp.name }}
+  {{- if .Values.collectors.gobmp.service.annotations }}
+  annotations:
+  {{- toYaml .Values.collectors.gobmp.service.annotations | nindent 4 }}
+  {{- end }}
 spec:
+  {{- if (or (eq .Values.collectors.gobmp.service.type "ClusterIP") (empty .Values.collectors.gobmp.service.type)) }}
+  type: ClusterIP
+  {{- with .Values.collectors.gobmp.service.clusterIP }}
+  clusterIP: {{ . }}
+  {{- end }}
+  {{- else if eq .Values.collectors.gobmp.service.type "LoadBalancer" }}
+  type: LoadBalancer
+  {{- with .Values.collectors.gobmp.service.loadBalancerIP }}
+  loadBalancerIP: {{ . }}
+  {{- end }}
+  {{- else }}
+  type: {{ .Values.collectors.gobmp.service.type }}
+  {{- end }}
   ports:
   - name: incbmpsess
```

Suggest a meaningful name
```suggestion
  - name: incomingBMPSessions
```